src/lib/Hydra/Plugin/GithubStatus.pm

```diff
@@ -26,7 +26,7 @@ sub toGithubState {
 }
 
 sub common {
-    my ($self, $topbuild, $dependents, $finished) = @_;
+    my ($self, $topbuild, $dependents, $finished, $onlyEval) = @_;
```

Comment:
It's not immediately obvious to me what `onlyEval` is actually for -- maybe this should be `cachedEval`, since it looks like it's only used from `cachedBuild*`?

Reply:
:+1: Applied, thanks!